src/main/webapp/app/complaints/list-of-complaints/list-of-complaints.component.ts

```diff
@@ -87,7 +87,7 @@ export class ListOfComplaintsComponent implements OnInit {
             () => (this.loading = false),
         );
     }
-
+    /** opens the assessment editor of the corresponding exercise type*/
```

Comment:
I would prefer to have a space before the `*/`. This also applies to other occurrences.